.storybook/style/storybook-info-details.scss

```diff
@@ -0,0 +1,19 @@
+.storybook-info {
```

Comment:
We're trying to remove our dependency on `scss` in favour of `styled components`, so we don't want to add more scss to the repo at this stage.

Reply:
Updated now to use styled components.

I have, however, had to leave some `scss` in, where I needed to apply styles to the `#story-root` element, which we don't have direct access to any other way